### ISSUE TITLE
Rule set does not exist: `default-rule-set`

### ISSUE THREAD

**Issue by tingletech** [hybox/aws#75]
When I try to start up the stack, I'm getting this error when the mail stack tries to create a lambda function:

Rule set does not exist: default-rule-set

https://gist.github.com/tingletech/e0e09bf0fd584fd721424880a13cfe51

Any ideas how to proceed?

Thanks -- Brian

**Comment by tingletech** [comment]
I went to SES and started to create a new rule
<img width="692" alt="screen shot 2016-08-14 at 12 43 15 pm" src="https://cloud.githubusercontent.com/assets/227374/17651488/d9fecf5a-621c-11e6-8355-dcfd92310a64.png">

When I got a couple of pages in, I noticed "create new Rule set"
<img width="980" alt="screen shot 2016-08-14 at 12 43 56 pm" src="https://cloud.githubusercontent.com/assets/227374/17651486/d9f8f648-621c-11e6-816b-057826468445.png">

<img width="577" alt="screen shot 2016-08-14 at 12 44 12 pm" src="https://cloud.githubusercontent.com/assets/227374/17651487/d9fc38d0-621c-11e6-9cf7-2f9970458829.png">

so, now I have created a default-rule-set in SES in virginia. ... hopefully that will work

**Comment by tingletech** [comment]
creating that rule fixed it.

**Comment by cbeer** [comment]
This seems useful to fix; we can probably generate a new rule set in the template.

**Comment by pmather** [comment]
I stumbled across this problem, so I would recommend either the rule set is checked for/created in the template or else the documentation be amended to instruct the user explicitly to create it.

BTW, I was able to create it thus via the CLI:

```
aws --region $AWS_DEFAULT_REGION ses create-receipt-rule-set --rule-set-name default-rule-set
```

**Comment by eefahy** [comment]
Fixed with f631663f93f27c87a6e3551e222080ac72598d0c

**Comment by mjgiarlo** [comment]
Thanks, @tingletech @pmather!
